{
  "action": {
    "error": {
      "variety": [
        "Capacity shortage",
        "Programming error"
      ],
      "vector": [
        "Carelessness",
        "Inadequate technology"
      ]
    }
  },
  "actor": {
    "internal": {
      "motive": [
        "NA"
      ],
      "variety": [
        "Developer",
        "System admin"
      ]
    }
  },
  "asset": {
    "assets": [
      {
        "amount": 1,
        "variety": "S - Web application"
      }
    ],
    "cloud": [
      "Unknown"
    ],
    "country": [
      "IT"
    ],
    "role": [
      "IT"
    ],
    "total_amount": 1
  },
  "attribute": {
    "availability": {
      "variety": [
        "Degradation",
        "Interruption"
      ]
    },
    "confidentiality": {
      "data": [
        {
          "variety": "Personal"
        }
      ],
      "data_disclosure": "Yes",
      "data_victim": [
        "Customer"
      ],
      "state": [
        "Stored unencrypted"
      ]
    }
  },
  "confidence": "Medium",
  "discovery_method": {
    "external": {
      "variety": [
        "Customer"
      ]
    }
  },
  "impact": {
    "overall_rating": "Distracting"
  },
  "incident_id": "2c285ba0-0e5a-11eb-ad9a-2114263f926d",
  "plus": {
    "analysis_status": "Reviewed",
    "analyst": "gbassett",
    "asset_os": [
      "Other"
    ],
    "attribute": {
      "confidentiality": {
        "data_abuse": "No"
      }
    },
    "created": "2020-10-15T01:31:15.726Z",
    "dbir_year": 2021,
    "event_chain": [
      {
        "action": "err",
        "actor": "int",
        "asset": "srv",
        "attribute": "cp",
        "summary": "actions occurred in paralel"
      },
      {
        "action": "err",
        "actor": "int",
        "asset": "srv",
        "attribute": "au",
        "summary": "actions occurred in parallel"
      }
    ],
    "github": "15404",
    "master_id": "861a1221-bc31-472c-903d-f12e7ed30809",
    "modified": "2020-10-15T01:31:15.726Z"
  },
  "reference": "https://portswigger.net/daily-swig/inps-hack-italys-social-security-website-back-online-following-cyber-attack-claims",
  "schema_version": "1.3.6",
  "security_incident": "Confirmed",
  "source_id": "vcdb",
  "summary": "overwhelmed website suffers degredation and information leakage (likely through cached profiles)",
  "targeted": "NA",
  "timeline": {
    "compromise": {
      "unit": "Hours"
    },
    "containment": {
      "unit": "Hours"
    },
    "discovery": {
      "unit": "Minutes"
    },
    "exfiltration": {
      "unit": "Seconds"
    },
    "incident": {
      "day": 1,
      "month": 4,
      "year": 2020
    }
  },
  "victim": {
    "country": [
      "IT"
    ],
    "employee_count": "25001 to 50000",
    "government": [
      "Unknown"
    ],
    "industry": "923130",
    "locations_affected": 1,
    "region": [
      "150039"
    ],
    "state": "IT-62",
    "victim_id": "Istituto nazionale della previdenza sociale"
  }
}